.github/workflows/actions.yml

```diff
@@ -454,6 +454,8 @@ jobs:
             -DCMAKE_BUILD_TYPE=RelWithDebInfo \
             -DCMAKE_INSTALL_PREFIX=pfx \
             -DCMAKE_PREFIX_PATH=`brew --prefix`/opt/qt \
+            -DCMAKE_CXX_FLAGS='--system-header-prefix=/usr/local/include' `# Needed to supress system header warnings in macos-15-intel` \
```

Comment:
I _think_ this should be `--system-header-prefix=fmt/` rather than the absolute path?

Reply:
Nice. That worked.

Look at the warnings again, I really don't think that's an error on our part. All we did was taking the target from `fmt-config.cmake` then linked to it, so those headers file should have been labelled as system headers without `--system-header-prefix` or other workarounds. All other runners were fine too.